### ISSUE TITLE
Depth label should be able to ignore outliters 

### ISSUE THREAD

**Issue by danifpdra** [lardemua/atom#614]
![image](https://github.com/lardemua/atom/assets/47574905/9888ada7-0f1d-41d0-9fec-7cc4c188d724)

**Comment by miguelriemoliveira** [comment]
Hi @danifpdra ,

what do you mean? You mean the automatic labeling should be able to do this?

**Comment by danifpdra** [comment]
Yes, it should. It doesn't make much sense to have an automatic labeler if I have to manually fix all the collections later. And those outliers shouldn't be very difficult to identify.

**Comment by miguelriemoliveira** [comment]
Hi @danifpdra ,

I will try to work on this, but first I will create a simple rgbd camera system to put in atom_examples.

**Comment by miguelriemoliveira** [comment]
Hi @danifpdra ,

I created the rgb_depth_system. I think I solved the problem we had about the camera being a single chassis with two distinct sensors.

I cut the mesh in half and now we will be able to see a mesh for both  cameras. And if the calibration works fine we should have a nice visual feedback of both parts comming toghether.

Check https://github.com/lardemua/atom/tree/noetic-devel/atom_examples/rgb_depth_system


Will now try to calibrate it and solve this problem on the way.
